### PR TITLE
Multi Receiver Testbed Support

### PR DESCRIPTION
**Pull request by taj-p**
## Intent

Multi Receiver Testbed Support. 

See https://github.com/Canva/otel-platform/pull/46 for how this is used.

### CODE REVIEW

File: .chloggen/res-attr-dont-drop-system-attributes.yaml
Comment (taj-p) [inline .chloggen/res-attr-dont-drop-system-attributes.yaml:1]: ignore this - this is a mistake

**Comment by taj-p** [comment]
@foadnh did you have time to review this?

**Comment by foadnh** [comment]
Oh sorry, I forgot about this PR! Checking it now.

**Review by foadnh — APPROVED** [review]
LGTM, I assume a good portion of codes are copied and modified versions of single receivers.

**Pull request merged**